app/components/chips_component.rb

```diff
@@ -0,0 +1,9 @@
+class ChipsComponent < ViewComponent::Base
+    def initialize(text:)
```

Comment:
text is not meaningful,  is it the value

Reply:
I think that it would be better to have two arguments `name` and `value'. But the `name` will have a default value which is `text_underscore`